lib/results.js

```diff
@@ -3,6 +3,7 @@ var inherits = require('inherits');
 var through = require('through');
 var resumer = require('resumer');
 var inspect = require('object-inspect');
+var has = Object.prototype.hasOwnProperty;
```

Comment:
The benefit of adding the bind is that `Function.prototype.call` could be removed/broken, and this module wouldn't break. It's more robustness than most bother with, though :-)